aspnetcore/fundamentals/logging/index/samples/3.x/MyMain/Program.cs

```diff
@@ -18,10 +17,11 @@ public static void Main(string[] args)
         {
             CreateHostBuilder(args).Build().Run();
         }
+
         public static IHostBuilder CreateHostBuilder(string[] args) =>
             Host.CreateDefaultBuilder(args)
-                .ConfigureLogging(log =>
-                   log.AddFilter("System", LogLevel.Debug)
+                .ConfigureLogging(logging =>
```

Comment:
`logging` is referenced in the topic right underneath this, so I changed that to make the reference correct. I also updated  other places where it was different for total consistency.